lib/directory.coffee

```diff
@@ -159,8 +159,12 @@ class Directory
       names = fs.readdirSync(@path)
     catch error
       names = []
-    names.sort (name1, name2) -> name1.toLowerCase().localeCompare(name2.toLowerCase())
-    
+
+    if atom.config.get('tree-view.alphanumericSort')
```

Comment:
Should the option be called `naturalSort` instead? Sort is *always* alphanumeric isn't it? The difference here is the natural part.